basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/LoggingAbstractTestCase.java

```diff
@@ -0,0 +1,117 @@
+package org.jboss.hal.testsuite.test.configuration.logging;
+
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.dmr.ModelNode;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.creaper.ResourceVerifier;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.page.config.LoggingPage;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.wildfly.extras.creaper.commands.logging.AddPeriodicRotatingFileLogHandler;
+import org.wildfly.extras.creaper.commands.logging.Logging;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Jan Kasik <jkasik@redhat.com>
+ *         Created on 7.12.15.
+ */
+public abstract class LoggingAbstractTestCase {
+
+    @Page
+    private LoggingPage page;
+
+    protected static Address LOGGING_SUBSYSTEM;
+
+    protected static OnlineManagementClient client;
+    protected static Operations operations;
+    protected static Administration administration;
+
+    @BeforeClass
+    public static void beforeClass_() {
+        client = ManagementClientProvider.createOnlineManagementClient();
+        operations = new Operations(client);
+        administration = new Administration(client);
+        LOGGING_SUBSYSTEM = client.options().isStandalone ? Address.subsystem("logging") :
+                Address.of("profile", client.options().defaultProfile).and("subsystem", "logging");
```

Comment:
Well, it turned out that it was my misunderstanding... I thought that if `Address` is processed with `Operations` the adjustment to domain/standalone is not done.
The reason why this happened is funny. We are using our own `ResourceVerifier` and it throws an exception with error message which contains address without adjustment and this confused me. The problem was obviously caused by something else, but I forgot about this in the end, sorry.
I "fixed" this in another pull request...